jenkins/Jenkinsfile-blossom.premerge-databricks

```diff
@@ -138,7 +140,22 @@ void databricksBuild() {
     def CLUSTER_ID = ''
     def SPARK_MAJOR = BASE_SPARK_VERSION_TO_INSTALL_DATABRICKS_JARS.replace('.', '')
     def dbStep = ''
+    def pluginBuiltTar ="$PLUGIN_BUILT_DIR/$DB_RUNTIME/spark-rapids-built.tgz"
+    // Map DBFS path to the local path into the cluster
+    def buildArgs = (params.TEST_MODE == 'CI_PART1') ? pluginBuiltTar.replace('dbfs:/', '/dbfs/') : ''
+    def testArgs = (params.TEST_MODE == 'CI_PART2') ?  pluginBuiltTar.replace('dbfs:/', '/dbfs/') : ''
+
     try {
+        // wait for all the rapids plugin tars built in CI_PART1 to be ready
+        if (params.TEST_MODE == 'CI_PART2') {
+            // Check if the comma-separated files exist in the Databricks DBFS path within timeout minutes
+            if (DbUtils.filesExist(this, "$pluginBuiltTar", 60)) {
+                println('Rapids plugin built tars are ready for CI_PART2')
+            } else {
+                error "Timeout, rapids plugin built tars are not ready for CI_PART2"
```

Comment:
I am not sure if we should downgrade to run build within CI2 if timeout. 

But we leave a comment here to remind us if we would like to try other options

Reply:
Sounds reasonable,

Updated to build rapids plugin if CI2 timeout waiting for the file spark-rapids-built.tgz